### ISSUE TITLE
Panada 2.0 for command line based application

### ISSUE THREAD

**Issue by cakyus** [panada/app#3]
Dear Panada Developers,

I work heavily in command line based application. Did you intended to develop panada framework only for web application ?  I already cloned this repo and founds error coz currently this code seems does not aware that application being executed in command line. If you welcome command line coders, i would be more than happy to join the development.

**Comment by ikandars** [comment]
Yes, Panada 2 will have CLI feature. You are very welcome to contribute to Panada 2 development. So please push your code and we will discussed it.

As an overview, it looks interesting if we can integrate with https://github.com/webmozart/console

**Comment by cakyus** [comment]
0kay , talking about webmozart/console .. i would prefer to use something like "Panada\Console\Uri" .. so, once a developer learn how "Panada\Http\Uri" works and he wants to switch to command line, they dont have to learn whole concept of webmozart/console ..
